pkgs/applications/editors/neovim/default.nix

```diff
@@ -32,6 +32,7 @@ let
       luaPackages.lua
       gperf
     ] ++ optional withJemalloc jemalloc
+      ++ optional stdenv.isDarwin libiconv
```

Comment:
This is a blind fix. Let's see if it works.

Reply:
Note that we almost always want to unconditionally add libiconv. Things like musl will also need it as well.